pyseir/inference/infer_rt.py

```diff
@@ -378,8 +486,19 @@ def get_posteriors(self, timeseries_type, plot=False):
         # of the data for maximum likelihood calculation.
         log_likelihood = 0.0
 
+        # Initialize scaling (for auto sigma)
+        scale = timeseries.head(1).item()
+
+        # Setup monitoring for Reff lagging signal in daily likelihood
+        monitor = LagMonitor(debug=False)  # Set debug=True for detailed printout of daily lag
+
         # (5) Iteratively apply Bayes' rule
         for previous_day, current_day in zip(timeseries.index[:-1], timeseries.index[1:]):
+
+            # Calculate process matrix for each day
+            scale = 0.9 * scale + 0.1 * timeseries[current_day]
```

Comment:
Comment

Reply:
Sure will  explain that this is an exponential moving average